addons/mail/models/res_users_settings_volumes.py

```diff
@@ -22,6 +22,9 @@ def init(self):
         ("partner_or_guest_exists", "CHECK((partner_id IS NOT NULL AND guest_id IS NULL) OR (partner_id IS NULL AND guest_id IS NOT NULL))", "A volume setting must have a partner or a guest."),
     ]
 
+    def name_get(self):
+        return [(rec.id, '%s - %s' % (rec.user_setting_id.user_id.name, rec.partner_id.name or '')) for rec in self]
```

Comment:
You may use fstrings, easier when reading code :)

Reply:
updated with fstring :)